htdocs/inc.loadControls.php

```diff
@@ -0,0 +1,175 @@
+<?php
+include ("lang/lang-en-UK.php");
```

Comment:
Hi  @hinnerkoetting this line makes me wonder if and how the ajax version should handle the interface language - which can be switched.

Reply:
@MiczFlor I "solved" it by printing all localisations in the index.php, so that they can be accessed by javascript.
I guess it is not a perfect solution but should be good enough for now in my opinion.